### DISCUSSION TITLE
CIPP not showing any Tenants

### DISCUSSION THREAD

**Discussion by protek-support** [KelvinTegelaar/CIPP#705] [General]
I've been using the CIPP for a couple of months now and as of a couple of days ago its no longer showing any Tenants. We haven't made any changes to things since I last updated it to the latest version. It was working fine after the last update.

If I run a permissions check I get the following error. "TypeError: Cannot read properties of undefined (reading 'map')"

Any help would be greatly appreciated.

**Comment by gregorykcts** [comment]
Let me jump onto this one instead of opening a new thread; I'm getting the same thing since a few days ago; I've redeployed the app, and its showing:

"Error loading tenants"

And on perms check it outputs:

Something went wrong.
TypeError: t.filter is not a function

I@https://FAKEURL.COM/static/js/main.f4efe485.js:2:11541
div
27998/Be<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:127356
div
27998/Ue<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:126980
div
27998/Wn<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:164921
div
27998/Jn<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:166468
div
g@https://FAKEURL.COM/static/js/8666.40c8898e.chunk.js:2:6801
div
t@https://FAKEURL.COM/static/js/main.f4efe485.js:2:110396
27998/va<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:204394
div
27998/ba<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:204130
div
27998/Be<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:127356
div
27998/Ue<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:126980
div
u@https://FAKEURL.COM/static/js/3423.accc706e.chunk.js:1:393
b@https://FAKEURL.COM/static/js/8666.40c8898e.chunk.js:2:5883
Suspense
le@https://FAKEURL.COM/static/js/main.f4efe485.js:2:40075
u@https://FAKEURL.COM/static/js/main.f4efe485.js:2:2147174
Suspense
div
27998/Xn<@https://FAKEURL.COM/static/js/main.f4efe485.js:2:166003
div
div
div
M@https://FAKEURL.COM/static/js/6971.8a089e12.chunk.js:1:6571
le@https://FAKEURL.COM/static/js/main.f4efe485.js:2:40075
f@https://FAKEURL.COM/static/js/main.f4efe485.js:2:2147933
Q@https://FAKEURL.COM/static/js/main.f4efe485.js:2:36234
Suspense
p@https://FAKEURL.COM/static/js/main.f4efe485.js:2:2147433
u@https://FAKEURL.COM/static/js/main.f4efe485.js:2:2144560
h
t@https://FAKEURL.COM/static/js/main.f4efe485.js:2:2246307
u@https://FAKEURL.COM/static/js/main.f4efe485.js:2:2141689

**Comment by gregorykcts** [comment]
*Error is on access check, while perms check gives "Your Secure Application Model has all required permissions".

**Comment by protek-support** [comment]
A new version just came out and I updated to it but that didn't fix my issue.

I'm see the exact same errors as you Greg.

**Reply by KelvinTegelaar** [reply]
You most likely are having token issues. Please follow *all* instructions on https://cipp.app and execute the test script. It could be anything from condtional access to tenants without licenses for what your trying.

**Comment by gregorykcts** [comment]
After some additional time (I've just rebuilt the app) it started showing up tenants in the app.
@protek-support  One of the things I've noticed is that my application secret has expired on the secure app i was using - I am not sure if that is/was part of the problem.

**Comment by gregorykcts** [comment]
@KelvinTegelaar  I've got some tenants with condtional access policies, that seem to be blocking access from CIPP - any quick guidance how this could be allowed, what kind of whitelist can be implemented (I did not yet pinpoint which exact cond policy is blocking this - i got policies such as MFA enforcement, legacy auth blocks and similar)?

Thank you.

**Reply by gregorykcts** [reply]
Sample of error code on one of the inaccessible tenants:
Any way to get some better error info?

Request failed with status code 500

{
  "status": 500,
  "data": "",
  "message": "Request failed with status code 500"
}

**Comment by protek-support** [comment]
I ended up recreating mine as well and its working now. Thanks for trying to help, I really appreciate it.
